web/.dockerignore

```diff
@@ -0,0 +1,4 @@
+**/node_modules
+
+alerting
```

Comment:
these are not needed anymore with this folder structure

Reply:
True